app/views/users/_notifications.html.erb

```diff
@@ -62,6 +62,16 @@
       <%= f.label :mobile_comment_notifications, "Notify me when someone replies to me in a comment thread" %>
     </div>
   </div>
+  <h3>Platform Notification Settings</h3>
+  <p>
+    <em>Notifications that only appear on the <a href="/notifications">notifications page</a>.</em>
+  </p>
+  <div class="checkbox-field">
+    <div class="sub-field">
+      <%= f.check_box :welcome_notifications %>
+      <%= f.label :welcome_notifications, "Send me occasional tips and tricks" %>
```

Comment:
Quick non-blocking nitpick: I feel like this copy here doesn't quite convey what the User is opting into or what Welcome Notifications are for. What if we had something like "Send me occasional tips on how to enhance my DEV experience" or something more along the lines of that?

Reply:
Yeah, I don't love the copy I had and was hoping people would have ideas on something better. I can certainly update, but I'd also like @lisasy to weigh in here.

This whole page could generally be re-worked for copy, I think; the repeated "send me email" has been making my eyes twitch over here 👀

Reply:
I like @juliannatetreault's revision to this — it's clear on the cadence and on what the tips will be for.